improbable/dockerized_tests/bin/create_bazel_cache_rcs.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
```

Comment:
This file is lifted almost entirely from `images/bootstrap`, using the K8S org's cache-key algorithm.  Good enough for them, good enough for us.

The specific changes I made here are:

1. Use our GCS HTTPS cache instead of their cluster-local cache server.
1. Hardcode some components of the cache-id since we don't need to use this image to support multiple repos like they do, and it's clearer.